compiler/src/iree/compiler/InputConversion/Common/CMakeLists.txt

```diff
@@ -8,8 +8,32 @@
 # To disable autogeneration for this file entirely, delete this header.        #
 ################################################################################
 
+# Enable input dialects based on options.
+set(IREE_INPUT_DEPS "")
+if(IREE_INPUT_MHLO)
```

Comment:
I think this file is supposed to be auto-generated.... I dont think the bazel changes + bazel_to_cmake is consistent with the changes in this file.

Reply:
Changed header so that it isn't autogenerated (this is following Pipeline.cpp structure).

Reply:
These changes do make sense, though we could restructure a bit if we want to keep it autogenerated. (I'm fine with just leveraging autogeneration where it helps and making manual changes like this as needed)